src/Compilers/Core/Portable/Operations/ControlFlowGraph.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    /// <summary>
+    /// PROTOTYPE(dataflow): Add documentation
```

Comment:
❔ Do we have an analyzer capable of reporting warnings on this? If not, we could switch to the more standardized approach of using `<placeholder>` for which analyzers [are available](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1651.md).

Reply:
@sharwell We don't need an analyzer for that. This is our standard way of marking incomplete things in feature branches. All comments like that will have to be addressed before we declare the feature complete. I am not interested in getting any diagnostics about them.